src/applications/mhv-medical-records/util/pdfHelpers/labsAndTests.js

```diff
@@ -11,6 +11,11 @@ export const generateChemHemContent = record => ({
   details: {
     header: 'Details about this test',
     items: [
+      {
+        title: 'Date and time performed',
```

Comment:
Is this field supposed to be "Date and time collected" or "Date and time performed"? It's different on PDF and web. Let's figure out which it's supposed to be and make it consistent.

Based on John's mapping it looks like it's supposed to be "Date and time collected".

https://fantastic-barnacle-35y3ge8.pages.github.io/StructureDefinition-VA.MHV.PHR.chTest-mappings.html